### PR TITLE
Event Publisher NATS Helm Chart

### PR DESCRIPTION
**Pull request by montaro**
**Related issue(s)**
See also #9852

### CODE REVIEW

File: resources/eventing/charts/event-publisher-nats/values.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 32m
+    memory: 64Mi
+
+nats:
+  config:
+   port: "8080"
+   natsPublishURL: nats.nats.svc.cluster.local
```
Comment (montaro) [inline resources/eventing/charts/event-publisher-nats/values.yaml:31]: To be more parameterized

File: resources/eventing/charts/event-publisher-nats/templates/clusterrole.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{{- if not .Values.global.isBEBEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "event-publisher-nats.fullname" . }}
+  labels: {{- include "event-publisher-nats.labels" . | nindent 4 }}
```
Comment (marcobebway) [inline resources/eventing/charts/event-publisher-nats/templates/clusterrole.yaml:6]: +1 for using `nindent`.

**Review by marcobebway — APPROVED** [review]

File: installation/resources/installer-cr-cluster.yaml.tpl [hunk 3]
```diff
@@ -19,8 +19,8 @@ spec:
       namespace: "istio-system"
```
Comment (marcobebway) [inline installation/resources/installer-cr-cluster.yaml.tpl:19]: Why do we need to change this file?

**Review by marcobebway — CHANGES_REQUESTED** [review]

File: installation/resources/installer-cr-cluster.yaml.tpl (hunk 3 above)
Comment (montaro) [reply installation/resources/installer-cr-cluster.yaml.tpl:19]: pushed by mistake, will revert

**Review by marcobebway — APPROVED** [review]

**Pull request merged**